src/applications/vaos/tests/new-appointment/components/ClinicChoicePage.unit.spec.jsx

```diff
@@ -489,6 +489,7 @@ describe('VAOS <ClinicChoicePage>', () => {
     await waitFor(
       () => expect(screen.getByLabelText(/red team/i).checked).to.be.true,
     );
+    userEvent.click(screen.getByRole('button', { name: /Continue/i }));
```

Comment:
We don't save form data until changing the page with the form state hook, so this test needs to have a continue button click.